performance-tools/performance-cluster/Dockerfile

```diff
@@ -64,6 +64,19 @@ RUN echo "deb https://dl.bintray.com/loadimpact/deb stable main" | tee -a /etc/a
 RUN apt-get update
 RUN apt-get install k6
 
+# Install Helm
+ENV HELM_VERSION=v2.0.0
+
+ENV HELM_FILE_NAME helm-${HELM_VERSION}-linux-amd64.tar.gz
```

Comment:
Can we stick to one notation with ENV? Either `ENV key=value` or `ENV key value` ? Then it is consistent.